api/server/server.go

```diff
@@ -104,25 +104,21 @@ type Config struct {
 
 // NewServer starts and returns a new server with the given configuration.
 func NewServer(conf Config) (*Server, error) {
-	var c *apistruct.FullNodeStruct
-	var cls func()
 	var err error
 	var masterAddr address.Address
+	c, cls, err := lotus.New(conf.LotusAddress, conf.LotusAuthToken)
+	if err != nil {
+		return nil, fmt.Errorf("connecting to lotus node: %s", err)
+	}
+
 	if conf.Embedded {
-		c, cls, err = lotus.NewEmbedded()
-		if err != nil {
-			return nil, fmt.Errorf("creating the embedded network: %s", err)
-		}
-		masterAddr, err = c.WalletDefaultAddress(context.Background())
-		if err != nil {
-			return nil, fmt.Errorf("getting default address: %s", err)
+		if masterAddr, err = c.WalletDefaultAddress(context.Background()); err != nil {
```

Comment:
In _embedded_ mode we get the _master address_ as the default, like in the _old_ days.
This because that whole devnet is ephemeral.